src/widgets/dialogs/LoginDialog.cpp

```diff
@@ -238,15 +233,15 @@ void AdvancedLoginWidget::refreshButtons()
     }
 }
 
-LoginWidget::LoginWidget(QWidget *parent)
+LoginDialog::LoginDialog(QWidget *parent)
     : QDialog(parent)
 {
-#ifdef USEWINSDK
-    ::SetWindowPos(HWND(this->winId()), HWND_TOPMOST, 0, 0, 0, 0,
-                   SWP_NOMOVE | SWP_NOSIZE | SWP_SHOWWINDOW);
-#endif
+    this->setMinimumWidth(300);
+    this->setWindowFlags(
+        (this->windowFlags() & ~(Qt::WindowContextHelpButtonHint)) |
+        Qt::Dialog | Qt::MSWindowsFixedSizeDialogHint);
 
-    this->setWindowTitle("Chatterino - add new account");
+    this->setWindowTitle("Add new account");
```

Comment:
This in 2nd PR as well

Reply:
this change is in scope - window now acts as modal dialog, these should only use feature name in caption when initiated by user action